debian/changelog

```diff
@@ -1,6 +1,10 @@
 matrix-synapse-py3 (1.47.0+nmu1) UNRELEASED; urgency=medium
 
   * Update scripts to pass Shellcheck lints.
+  * Remove unused Vagrant scripts from debian/ directory.
```

Comment:
well, *I* used them occasionally :/.

Probably not often enough to insist you reinstate them though.

Reply:
They were just for Stretch and Xenial, neither of which we support anymore :(

Reply:
no, but they could be updated more easily than reconstructing them from scratch. nm.